src/ZoneServer/Network/Send.cs

```diff
@@ -3236,15 +3275,23 @@ public static void ZC_PCBANG_POINT(IZoneConnection conn)
 		}
 
 		/// <summary>
-		/// Updates character's movement speed.
+		/// Updates entity's movement speed.
 		/// </summary>
-		/// <param name="character"></param>
+		/// <param name="entity"></param>
 		public static void ZC_MSPD(ICombatEntity entity)
+			=> ZC_MSPD(entity, entity);
+
+		/// <summary>
+		/// Updates entity movement speed for another entity that has connection.
+		/// </summary>
+		/// <param name="entity"></param>
+		/// <param name="target"></param>
+		public static void ZC_MSPD(ICombatEntity entity, ICombatEntity target)
```

Comment:
Seeing all these sender overloads, I'm starting to wonder whether dummies perhaps _should_ be able to receive packets. If they were forwarded to their owner's client, you would potentially not need any of these.